default.json

```diff
@@ -1,5 +1,11 @@
 {
   "display": {
+    "annotations": {
```

Comment:
Yes, and I'm asking whether it's okay [here](https://github.com/SassDoc/sassdoc-theme-default/pull/58#issuecomment-72336760). :)